### PR TITLE
Adding GalaxyStaking

### PR DESCRIPTION
**Pull request by fernando-galaxystaking**

### CODE REVIEW

File: src/whois/crypto.org/others/GalaxyStaking.space/index.ts [hunk 1]
```diff
@@ -5,4 +5,16 @@ export default <WhoisRecords>{
     label: 'GalaxyStaking.space Relayer',
     link: 'https://galaxystaking.space',
   },
+  cro1fvedw32vg68kv83s6zzgh4p06gydvf78t2z0cr: {
+    label: 'GalaxyStaking.space - Neptune Restake',
```
Comment (web4ken) [inline src/whois/crypto.org/others/GalaxyStaking.space/index.ts]: short names are better, can we use `GalaxyStaking – Neptune Restake`?

File: src/whois/crypto.org/others/GalaxyStaking.space/index.ts [hunk 2]
```diff
@@ -5,4 +5,16 @@ export default <WhoisRecords>{
...
+    link: 'https://galaxystaking.space',
+  },
+  cro19ewjs3w6r0frmgljhdts70hz9wlcyrfx2wed27: {
+    label: 'GalaxyStaking.space - Saturn Restake',
```
Comment (web4ken) [inline src/whois/crypto.org/others/GalaxyStaking.space/index.ts]: short names are better, can we use `GalaxyStaking – Saturn Restake`?

File: src/whois/crypto.org/others/GalaxyStaking.space/index.ts [hunk 3]
```diff
@@ -5,4 +5,16 @@ export default <WhoisRecords>{
...
+    link: 'https://galaxystaking.space',
+  },
+  cro1lemc6jwfny6ryjaf37w62a6xdpttrr0t0var3n: {
+    label: 'GalaxyStaking.space - Uranus Restake',
```
Comment (web4ken) [inline src/whois/crypto.org/others/GalaxyStaking.space/index.ts]: short names are better, can we use `GalaxyStaking – Uranus Restake`?

File: src/whois/crypto.org/others/GalaxyStaking.space/index.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { WhoisRecords } from '../../../../interfaces'
+
+export default <WhoisRecords>{
+  cro1lwymrvh7vpwzqfkgpvjtz2y5gqelw4m7tq0w3e: {
+    label: 'GalaxyStaking.space Relayer',
```
Comment (web4ken) [inline src/whois/crypto.org/others/GalaxyStaking.space/index.ts]: short names are better, can we use `GalaxyStaking Relayer`?

**Review by web4ken — CHANGES_REQUESTED** [review]
Also please make the directory in lowercase: `GalaxyStaking.space` -> `galaxystaking.space`

**Comment by fernando-galaxystaking** [comment]
Hi and thank you for reply. Here the recommended changes.

**Review by web4ken — APPROVED** [review]

**Pull request merged**